rc/core/expandtab.kak

```diff
@@ -0,0 +1,38 @@
+define-command -docstring "noexpandtab: use tab character to indent and align" \
+noexpandtab %{
+    remove-hooks global noexpandtab
+    hook -group noexpandtab global NormalKey <gt> %{ try %{
+        execute-keys -draft "<a-x>s^\h+<ret><a-@>"
```

Comment:
Why do we need this hook ? Is setting `indentwidth` to 0 not working ? (Maybe its not that well documented, I see that the info box does not mention `indentwidth` of 0 means using a tab character for indentation).

Reply:
> Is setting indentwidth to 0 not working ?

I didn't knew that. In this case this certainly can be used instead of hook.

> Why do we need this hook ? 

However how to know what `indentwidth` to set when we turn it back by calling another command, like `expandtab`? I surely can store the old value in a hidden option, however it'll be more complex compared to this hook.